go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/jmespath/jmespath
```

Comment:
I don't think this is right, shouldn't this be `github.com/jmespath/go-jmespath`

Reply:
@reedobrien please fix the module name